tests/unit/mixins/css-test.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Unit test for the CssMixin
+ */
+import {expect} from 'chai'
+import Ember from 'ember'
+const {Component} = Ember
+import PropTypesMixin from 'ember-prop-types'
+import {afterEach, beforeEach, describe, it} from 'mocha'
+import sinon from 'sinon'
+
+import CssMixin from 'ember-frost-core/mixins/css'
+
+const CssComponent = Component.extend(PropTypesMixin, CssMixin, {
+  toString () {
+    return '<app-name@component:component-name::ember555>'
```

Comment:
Any way we can pull this off a real component so that we catch any changes in Ember with a break in our tests?